{{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/urls.py

```diff
@@ -13,28 +13,17 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.conf import settings
 from django.contrib import admin
```

Comment:
are we sure to delete django_toolbar? silk was looking promising but it failed to deliver what we needed in the pidash case

Reply:
What would DDT deliver in addition to silk?